models/stock_warehouse.py

```diff
@@ -89,5 +99,7 @@ def _prepare_info(self):
             'u_print_labels_picking_type_ids': self.u_print_labels_picking_type_ids.ids,
             'u_pallet_barcode_regex': self.u_pallet_barcode_regex,
             'u_package_barcode_regex': self.u_package_barcode_regex,
+            'u_pi_count_move_picking_type': self.u_pi_count_move_picking_type.id,
```

Comment:
We have to add this field to the stock.warehouse udes_api README because we are now returning them.

https://github.com/unipartdigital/udes_api#stock-warehouse

Reply:
Same for u_stock_investigation_picking_type

Reply:
👍